src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo, useEffect } from 'react';
+import {
+  Container,
+  CloseButton,
+  Spinner,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { useParams } from 'react-router-dom';
+import messages from './messages';
+import ContentTagsCollapsible from './ContentTagsCollapsible';
+import { extractOrgFromContentId } from './utils';
+import {
+  useContentTaxonomyTagsDataResponse,
+  useIsContentTaxonomyTagsDataLoaded,
+  useContentDataResponse,
+  useIsContentDataLoaded,
+} from './data/apiHooks';
+import { useTaxonomyListDataResponse, useIsTaxonomyListDataLoaded } from '../taxonomy/data/apiHooks';
+import Loading from '../generic/Loading';
+
+const ContentTagsDrawer = () => {
+  const intl = useIntl();
+  const { contentId } = useParams();
+
+  const org = extractOrgFromContentId(contentId);
+
+  const useContentData = () => {
+    const contentData = useContentDataResponse(contentId);
+    const isContentDataLoaded = useIsContentDataLoaded(contentId);
+    return { contentData, isContentDataLoaded };
+  };
+
+  const useContentTaxonomyTagsData = () => {
+    const contentTaxonomyTagsData = useContentTaxonomyTagsDataResponse(contentId);
+    const isContentTaxonomyTagsLoaded = useIsContentTaxonomyTagsDataLoaded(contentId);
+    return { contentTaxonomyTagsData, isContentTaxonomyTagsLoaded };
+  };
+
+  const useTaxonomyListData = () => {
+    const taxonomyListData = useTaxonomyListDataResponse(org);
+    const isTaxonomyListLoaded = useIsTaxonomyListDataLoaded(org);
+    return { taxonomyListData, isTaxonomyListLoaded };
+  };
+
+  const { contentData, isContentDataLoaded } = useContentData();
+  const { contentTaxonomyTagsData, isContentTaxonomyTagsLoaded } = useContentTaxonomyTagsData();
+  const { taxonomyListData, isTaxonomyListLoaded } = useTaxonomyListData();
+
+  const closeContentTagsDrawer = () => {
+    // "*" allows communication with any origin
+    window.parent.postMessage('closeManageTagsDrawer', '*');
```

Comment:
I don't there is any set standard for this, but a few other places this is used it's a string like: 'lmshtmlfragment.resize' and 'tools.toggleNotes'.

Reply:
Make this change would require also making changes to the edx-platform. I think so it doesn't get blocked by another review cycle there, we can address it in a later PR.